### ISSUE TITLE
Visual Studio minor version tag

### ISSUE THREAD

**Issue by agat366** [mayerwin/vs-customize-window-title#34]
Hi!

It's really nice extension!
However, what is really lacking to me is the IDE minor version.
The thing is that IMHO `[vsMajorVersion]` is not quite useful as it's quite rare when you work with different Visual Studio major versions at the same time (like 2015 and 2017), however, it is really handy to have minor version currently installed on your system (like 15.7.1) as these days VS updates very often and you always can see if you need to upgrade or not.

So, can you consider adding that as well?
Thanks!

**Comment by mayerwin** [comment]
Thanks for the feature request. Not sure if it is easy to implement as the internal version number is different from the public one (e.g. VS 2017 = VS15). However, isn't it sufficient to see the notification that you need to upgrade (which are pushed reliably by Microsoft)?

**Comment by agat366** [comment]
This is just what I am sharing you from my experience of daily use of VS. Microsoft updates VS pretty often these days, and sometime I just get lost which version I've recently upgraded it to. Yes, I can go to "About..." and have a look at it, but this is just a question of convenience, right.
This might be not a bit thing (especially if the minor version detection is hard), but being a Xamarin developer I have to upgrade it often and look at new updates coming, so that might be not so vital for other developers. (Even though Core devs might also need to be on the pulse of VS updates).
From other hands I don't actually even remember when I launched 2015th version (and it's even less important to me what major version that was), which might be handy for others.
I appreciate your work in any case and will be happy to provide any other feedback in future.
